packages/shared/components/inputs/UnitInput.svelte

```diff
@@ -1,36 +1,40 @@
 <script lang="ts">
     import { Dropdown2 } from 'shared/components'
-    import { ITokenMetadata } from '@core/wallet'
-    import { IDropdownItem, IotaUnit } from '@core/utils'
+    import { TokenMetadata, TokenStandard, getUnitFromTokenMetadata } from '@core/wallet'
+    import { IotaUnit, IDropdownItem } from '@core/utils'
```

Comment:
Feels like `IotaUnit` should be with the other token logic

Reply:
Out of scope of this PR to clean up location of token functions